.github/workflows/github-pages

```diff
@@ -0,0 +1,91 @@
+# This is a basic workflow to help you get started with Actions
+name: Deploy Sveltekit site to Pages
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  # Build job
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Detect package manager
+        id: detect-package-manager
+        run: |
+          if [ -f "${{ github.workspace }}/yarn.lock" ]; then
+            echo "manager=yarn" >> $GITHUB_OUTPUT
+            echo "command=install" >> $GITHUB_OUTPUT
+            echo "runner=yarn" >> $GITHUB_OUTPUT
+            exit 0
+          elif [ -f "${{ github.workspace }}/package.json" ]; then
+            echo "manager=npm" >> $GITHUB_OUTPUT
+            echo "command=ci" >> $GITHUB_OUTPUT
+            echo "runner=npx --no-install" >> $GITHUB_OUTPUT
+            exit 0
+          else
+            echo "Unable to determine package manager"
+            exit 1
+          fi
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+          cache: ${{ steps.detect-package-manager.outputs.manager }}
+      - name: Setup Pages
+        uses: actions/configure-pages@v3
+        with:
+          static_site_generator: sveltekit
```

Comment:
Misschien dat deze het al doet, misschien ook niet... ik moest voor mijn andere SvelteKit sites nog wel wat extra dingen regelen:

- `@sveltejs/adapter-static` installeren als devDependency ([docs link](https://kit.svelte.dev/docs/adapter-static))
- In `svelte.config.js` ([voorbeeld uit mijn andere project](https://github.com/pincrediball/pincrediball.com/blob/main/svelte.config.js)) nog de `adapter-static` configureren, en daar ook de "404" opzet instellen, en het `paths` deel instellen (wat afweek van de docs omdat ik op een TLD (evt met subdomein) host, in plaats van `user.github.io/reponame`
- een (leeg) `.nojekyll` bestandje aanmaken ([voorbeeld](https://github.com/pincrediball/pincrediball.com/blob/main/static/.nojekyll)) in `/static` om te zorgen dat GH Pages servers Svelte routing z'n werk laat doen
- de `CNAME` file die GitHub voor je genereert verplaatsen naar `/static` zodat hij meegebuild wordt door Sveltekit

Wellicht is dit allemaal overbodige info omdat de GH Action stappen ze afhandelen, maar 'better safe than sorry'. Verder ziet alles er logisch uit voor mij.